docs/guides/writing-a-feature-app.md

```diff
@@ -37,8 +49,33 @@ required Feature Services][feature-service-binder] to the dependent Feature App.
 
 ## `dependencies`
 
-Required Feature Services are declared with their ID and a [semver version
-range][semver], e.g. `{'acme:some-feature-service': '^2.0.0'}`.
+The `dependencies` map can contain two types of required dependencies:
+
+1. With `dependencies.featureServices` all required Feature Services are
+   declared. If one of those dependencies can't be fulfilled, the Feature App
+   won't be created. This means the Feature App can be sure that those
+   dependencies are always present when it is created.
+
+   Feature Service dependencies are declared with their ID as key, and a [semver
+   version range][semver] as value, e.g.
+   `{'acme:some-feature-service': '^2.0.0'}`.
+
+1. With `dependencies.externals` all required external dependencies are
+   declared. This may include [shared npm
+   dependencies][sharing-npm-dependencies] that are provided by the integrator.
+
+   External dependencies are declared with their module name as key, and a
+   [semver version range][semver] as value, e.g. `{react: '^16.7.0'}`.
+
+## `optionalDependencies`
```

Comment:
Perhaps a sentence would be good that Externals currently can not yet be declared as optional. But there is already an issue which can be tackled on demand.

Reply:
done